data/human/events.txt

```diff
@@ -136,6 +136,8 @@ event "war begins"
 		"required reputation" 100
 	planet "Martini"
 		description `Martini is a world of exotic beaches and perfect weather. The Republic's central bank and the galactic stock exchange were previously headquartered here, before a terrorist attack destroyed them and a large portion of the capital city as well.`
+		description `	Currently, a massive rescue and evacuation operation is going on in and around the capital to find and treat survivors and contain the radiation. As to be expected, almost every supply is running dangerously low, and what remains of Martini's government (many in the planetary government died in the blast) has called for civilian ships to help deliver goods and supplies for the operation.`
```

Comment:
```suggestion
		description `	Currently, a massive rescue and evacuation operation is going on in and around the capital to find and treat survivors and contain the radiation. As to be expected, almost every supply is running dangerously low, and what remains of Martini's government has called for civilian ships to help deliver goods and supplies for the operation.`
```

Reply:
Fair enough, but I still want to tell the player that most of Martini's government was wiped out.